src/components/theme/View/View.jsx

```diff
@@ -240,17 +255,25 @@ class View extends Component {
             <meta
               property="og:image"
               content={toPublicURL(
-                this.props.content.image?.scales?.large?.download,
+                this.props.content.opengraph_image.download ||
```

Comment:
this.props.content.opengraph_image?.download

Reply:
At least in our kitconcept_seo implementation we did not use image scales for uploaded opengraph images afaik. @tisto can oyu confirm this?

Reply:
We should always scale images. Please check what image size og expects and use the appropriate image scale.

Reply:
also consider you may not have the pengraph_image. This is because i suggested to add '?' after opengraph_image